src/equations/ideal_glm_mhd_2d.jl

```diff
@@ -68,18 +69,19 @@ An Alfvén wave as smooth initial condition used for convergence tests.
 function initial_condition_convergence_test(x, t, equations::IdealGlmMhdEquations2D)
     # smooth Alfvén wave test from Derigs et al. FLASH (2016)
     # domain must be set to [0, 1/cos(α)] x [0, 1/sin(α)], γ = 5/3
-    alpha = 0.25 * pi
+    RealT = eltype(x)
+    alpha = 0.25f0 * convert(RealT, pi)
     x_perp = x[1] * cos(alpha) + x[2] * sin(alpha)
-    B_perp = 0.1 * sin(2.0 * pi * x_perp)
-    rho = 1.0
+    B_perp = convert(RealT, 0.1) * sinpi(2 * x_perp)
```

Comment:
Could you please check whether the CI failures are caused by changes `sin(pi * ...)` -> `sinpi(...)` like here?

Reply:
Sure.